src/lib.rs

```diff
@@ -9,7 +9,8 @@
 
 // From rustc.
 extern crate arena;
-#[macro_use] extern crate rustc;
+#[macro_use]
+extern crate rustc;
```

Comment:
I'm mostly ambivalent about this change. I used the same-line `#[macro_use]` after seeing it in other projects including rustc. However, checking now, it's not even consistent within rustc.

On the other hand, I want to keep the list of crates alphabetical.